2nd-gen/packages/swc/components/asset/asset.css

```diff
@@ -1,91 +1,49 @@
-/**
+/*!
  * Copyright 2025 Adobe. All rights reserved.
+ *
  * This file is licensed to you under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License. You may obtain a copy
- * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * of the License at <http://www.apache.org/licenses/LICENSE-2.0>
  *
  * Unless required by applicable law or agreed to in writing, software distributed under
  * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
 
-:host {
-    --spectrum-asset-folder-background: var(
-        --highcontrast-asset-icon-background-color,
-        var(
-            --mod-asset-folder-background-color,
-            var(--spectrum-asset-folder-background-color)
-        )
-    );
-    --spectrum-asset-file-background: var(
-        --highcontrast-asset-icon-background-color,
-        var(
-            --mod-asset-file-background-color,
-            var(--spectrum-asset-file-background-color)
-        )
-    );
-    --spectrum-asset-folder-outline: var(
-        --mod-asset-icon-outline-color,
-        var(--spectrum-asset-icon-outline-color)
-    );
-    --spectrum-asset-file-outline: var(
-        --mod-asset-icon-outline-color,
-        var(--spectrum-asset-icon-outline-color)
-    );
-
+.spectrum-Asset {
```

Comment:
The CSS here looks great- matches the S2 code we have in the `spectrum-two` branch. 👍 🥳

My only question is do we need a selector for `:host`, giving it the `display: flex` property? I believe the default display for `:host` is inline.

Reply:
Good catch @marissahuysentruyt ! Both display values serve different purposes and are both necessary.